src/13_file_io.py

```diff
@@ -9,11 +9,18 @@
 # Print all the contents of the file, then close the file
 # Note: pay close attention to your current directory when trying to open "foo.txt"
 
-# YOUR CODE HERE
+f = open("foo.txt", "r")
+print(f.read())
+f.close()
 
 # Open up a file called "bar.txt" (which doesn't exist yet) for
 # writing. Write three lines of arbitrary content to that file,
 # then close the file. Open up "bar.txt" and inspect it to make
 # sure that it contains what you expect it to contain
 
-# YOUR CODE HERE
\ No newline at end of file
+f = open("bar.txt", "w+")
+for i in range(3):
+    f.write("This is line %d\r\n" % (i+1))
+
```

Comment:
Nice use of `open()`. One downside is we have to manually close the file. We can use `with open()` which allows us to not have to explicitly close the file.